apps/financial-aid/web-osk/src/utils/useLogOut.ts

```diff
@@ -11,7 +11,12 @@ export const useLogOut = () => {
   const logOut = () => {
     api.logOut()
     setUser && setUser(undefined)
-    updateForm({ submitted: false, incomeFiles: [] })
+    updateForm({
+      submitted: false,
+      incomeFiles: [],
+      taxReturnFiles: [],
+      otherFiles: [],
+    })
```

Comment:
Should we have an initial state const and use that here and in form provider?

Reply:
It is in the other pull request, emptyFormProvider where I make a function where does exactly that :))